index.html

```diff
@@ -30,8 +30,20 @@
           </div>
         </div>
       </nav>
+      <!-- Starting mobile menu  -->
+      <div class="mobile-menu">
+        <div class="mobile-menu-header">
+        </div>
+        <ul class="mobile-menu-items">
+          <li><a href="#portfolio" class="mobile-header-color">Portfolio</a></li>
+          <li><a href="#about" class="mobile-header-color">About</a></li>
+          <li><a href="#contact" class="mobile-header-color">Contact</a></li>
```

Comment:
- [ ] As stated in the requirements, when the user clicks(taps) any of the mobile menu options, a correct part of the page is displayed. When I click on the #contact option, the contact section was not displayed. Let's kindly consider fixing this issue.

Reply:
Hey, I have fixed this issue